SixTrack/sixtrack.s

```diff
@@ -33161,30 +33260,44 @@ C     Convert r(1), r(2) from U(0,1) -> rvec0 as Gaussian with cutoff mcut (#sig
      &)
 !
 !++  Populate the efficiency arrays at the end of each turn...
+! Modified by M.Fiascaris, July 2016
 !
               if (ie.eq.iu) then
                 do ieff = 1, numeff
                   if (counted_r(j,ieff).eq.0 .and.                      &
 !GRD     &SQRT(NSPX**2+NSPY**2).GE.RSIG(IEFF)) THEN
      &sqrt(                                                             &
-     &((xineff(j)*1d-3)**2                                              &
+     &((xineff(j)*1d-3)**2 +                                            &
+     & (talphax(ie)*xineff(j)*1d-3 + tbetax(ie)*xpineff(j)*1d-3)**2)    &
      &/                                                                 &
-     &(tbetax(ie)*myemitx0))                                            &
+     &(tbetax(ie)*myemitx0)                                             &
      &+                                                                 &
-     &((yineff(j)*1d-3)**2                                              &
+     &((yineff(j)*1d-3)**2 +                                            &
+     & (talphay(ie)*yineff(j)*1d-3 + tbetay(ie)*ypineff(j)*1d-3)**2)    &
      &/                                                                 &
-     &(tbetay(ie)*myemity0))                                            &
+     &(tbetay(ie)*myemity0)                                             &
      &).ge.rsig(ieff)) then
```

Comment:
When writing the changelog / commit message when merging the pull, we should probably mention that this now includes the alpha? Since this may affect the results of other people who rely on counted_r and neff...